### PR TITLE
deps: bump Kubernetes Client to 6.7.1

### PR DESCRIPTION
**Pull request by manusa**
## Description

deps: bump Kubernetes Client to 6.7.1

Introduce internal JKube Serialization class to wrap around Kubernetes Client KubernetesSerialization serialization tooling.

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [x] My code follows the style guidelines of this project
 - [x] I Keep It Small and Simple: The smaller the PR is, the easier it is to review and have it merged
 - [x] I use [conventional commits](https://www.conventionalcommits.org/) in my commit messages
 - [x] I have performed a self-review of my code
 - [x] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [x] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Review by sunix — APPROVED** [review]

**Comment by manusa** [comment]
seems that the problem with the timeout is fixed now

File: jkube-kit/profile/src/test/java/org/eclipse/jkube/kit/profile/ProfileUtilTest.java [hunk 1]
```diff
@@ -32,29 +32,6 @@
...
-                .returns(true, c -> c.use("spring.swarm")));
-    }
-
-    @Test
-    void multiple() throws IOException {
-        InputStream is = getClass().getResourceAsStream("/jkube/config/ProfileUtilTest-multiple.yml");
-        assertThat(is).isNotNull();
-        List<Profile> profiles = ProfileUtil.fromYaml(is);
-        assertThat(profiles).hasSize(2);
-    }
```
Comment (rohanKanojia) [inline jkube-kit/profile/src/test/java/org/eclipse/jkube/kit/profile/ProfileUtilTest.java:56]: Why are these tests deleted? How is this change related to KubernetesClient upgrade?

Comment (rohanKanojia) [reply jkube-kit/profile/src/test/java/org/eclipse/jkube/kit/profile/ProfileUtilTest.java:56]: okay, I see you're removed ProfileUtil.fromYaml method

Comment (manusa) [reply jkube-kit/profile/src/test/java/org/eclipse/jkube/kit/profile/ProfileUtilTest.java:56]: Some of the tests were redundant and no longer applicable.

The remaining tests should cover all of the scenarios

**Review by rohanKanojia — APPROVED** [review]
